ts/Core/Axis/TreeGridAxis.ts

```diff
@@ -106,7 +106,7 @@ namespace TreeGridAxis {
      *
      * */
 
-    export interface AxisBreakObject extends AxisTypes.AxisBreakObject {
+    export interface AxisBreakObject extends Highcharts.XAxisBreaksOptions {
```

Comment:
We have to get away from referencing Highcharts namespace. In general it would make sense, to move all namespace types into DTS files, so that they become extendable like in a namespace.